libs/cypress-indexeddb/src/lib/object-store-metadata.ts

```diff
@@ -8,6 +8,16 @@ type ConsolePropObject = {
   error?: Error;
 };
 
+/**
+ * List keys in store
+ *
+ * @remarks The `keys` method lists the keys available in the provided store.
+ *
+ * @param store `IDBObjectStore` instance
+ *
+ * @returns IDBValidKey[]
+ * @throws {Error} If the connections fails to open or the read operation fails.
```

Comment:
technically it only throws an error if it is chained off from a method that does not return an object store.